### ISSUE TITLE
Massive CPU spikes when copy-pasting from alacritty into chromium-based software

### ISSUE THREAD

**Issue by guillaume-perreal** [alacritty/alacritty#6776]
Pasting the selection from alacritty into a chromium-based software causes a massive CPU spike of the later.  It happens when pasting using either the mouse button or the keyboard shortcuts. It causes the affected program to consume 100% of a single CPU and to freeze it for several (tens of) seconds. The selection is properly pasted at the end of the CPU spike.

It does not happen when copy-pasting: 
* from alacritty into alacritty,
* from alacritty into another software (e.g. firefox, gedit...),
* from another software into the chromium-based software.

### System

OS: Linux Mint Vera (21.1)
Version: alacritty 0.12.0-rc2 (from source using rustc 1.67.1 (d5a82bbd2 2023-02-07))
Linux: X11, i3-wm 4.22-2

Affected software:
* chromium 110.0.5481.100~linuxmint1+vera
* discord 0.0.25 (snap)
* mattermost-desktop 5.0.2-15005
* vscode 1.76.0

### Logs

Logs (alacritty -vv) :  https://pastebin.com/r2VKrnQx

**Comment by chrisduerr** [comment]
Considering this works everywhere other than chromium software, plus this is affecting chromium itself, what makes you think it's an Alacritty bug? Just that no other application is showing this?

**Comment by chrisduerr** [comment]
Also this is likely fixed in https://github.com/quininer/x11-clipboard/pull/34, so could you try the latest master?

**Comment by guillaume-perreal** [comment]
> Considering this works everywhere other than chromium software, plus this is affecting chromium itself, what makes you think it's an Alacritty bug? Just that no other application is showing this?

As I wrote, it only happens when copy-pasting specifically from alacritty into chromium (or any chromium-based software). I could not reproduce it with any other combination involving only one of either alacritty or chromium. For example, if I copy-paste from alacritty into gedit then from gedit into chromium, it works fine.

Though I am not sure if this is just a correlation or a cause, but it worked fine before I upgraded Linux Mint from Una (based on ubuntu focal) to Vera (based on ubuntu jammy) last week. It may have an issue.

However, it looks like this is a duplicate of #6760. I did not found it when I searched existing issues. Sorry.

> Also this is likely fixed in https://github.com/quininer/x11-clipboard/pull/34, so could you try the latest master?

Will do.
